build/config.example.py

```diff
@@ -1,12 +1,32 @@
+import os
 from pathlib import Path
 from vgm.pbo import PBO
 
-username = "Spoffy"
+username = os.getlogin()
+arma_profile = r"dev_vgm"
+
+paradigm_path = os.path.join(os.path.dirname(__file__), "../paradigm")
```

Comment:
Using pathlib would be an easier to read way of representing this, I think:
```suggestion
paradigm_path = Path(__file__).parent /  "paradigm"
```